src/src/com/microsoft/aad/adal/AuthenticationActivity.java

```diff
@@ -627,18 +626,17 @@ public boolean processInvalidUrl(final WebView view, String url) {
                         mRedirectUrl));
                 view.stopLoading();
                 return true;
-            }
-            
-            //check if the redirect URL is under SSL protected
-            if(!url.toLowerCase(Locale.US).startsWith(AuthenticationConstants.Broker.REDIRECT_SSL_PREFIX))
-            {
+            } else if (url.startsWith("about:blank")) {
+                Logger.v(TAG, "It is an blank page request");
+                return true;
+            } else if(!url.toLowerCase(Locale.US).startsWith(AuthenticationConstants.Broker.REDIRECT_SSL_PREFIX)){
```

Comment:
@weijjia  In this function, you treated SSL-protected page and "about:blank" as the same, where as a result, not only the ssl-protected page will be opened in the webview, but also the "about:blank" empty page. I am wondering in which situation we need to open an blank webpage. Should we override the "about:blank" url loading, or not? Thanks.